doc/fuzzing.md

```diff
@@ -75,10 +85,15 @@ $AFLPATH/afl-fuzz -i ${DIR_FUZZ_IN}/${FUZZ_TARGET} -o ${AFLOUT}/${FUZZ_TARGET} -
 You may have to change a few kernel parameters to test optimally - `afl-fuzz`
 will print an error and suggestion if so.
 
+On macOS you may need to set `AFL_NO_FORKSRV=1` to get the target to run.
+```
+AFL_NO_FORKSRV=1 $AFLPATH/afl-fuzz -i ${DIR_FUZZ_IN}/${FUZZ_TARGET} -o ${AFLOUT}/${FUZZ_TARGET} -m52 -- test/fuzz/${FUZZ_TARGET}
```

Comment:
I seem to be missing the ${FUZZ_TARGET} declaration. The comment below contains the two rows from the similar section at https://github.com/bitcoin/bitcoin/blob/master/doc/fuzzing.md

Reply:
done